src/content/docs/en/guides/configuring-astro.mdx

```diff
@@ -108,9 +108,9 @@ import { defineConfig } from 'astro/config'
 
 export default defineConfig({
   // Resolves to the "./foo" directory, relative to this config file
-  root: new URL("./foo", import.meta.url),
+  root: `${new URL("./foo", import.meta.url)}`,
```

Comment:
I think it might be nicer here to explicitly call `toString()` instead of using the template string to do that implicitly:

```suggestion
  root: new URL("./foo", import.meta.url).toString(),
```